tools/pipelines/templates/build-npm-client-package.yml

```
@@ -173,6 +173,8 @@ extends:
         arrow:
           # This is the service connection for the Arrow Service Connection in FluidFramework Azure DevOps organization
           serviceConnection: ff-internal-arrow-sc
+          # This will make sure that the artifacts are published to the Arrow Service Connection if they are release or pre-release
+          isShipped: ${{ if or(eq(variables['release'], 'release'), eq(variables['release'], 'prerelease')) }}true${{ else }}false
```

I don't know if single-line if-else like this works in ADO. You might just need to do:
```suggestion
        # Note: using `variables` in a compile-time condition is easy to get wrong; here we know that the 'release' variable does exist at compile time
        ${{ if or(eq(variables['release'], 'release'), eq(variables['release'], 'prerelease')) }}
          isShipped: true
```